### PR TITLE
ADR additional parameters

### LINKED ISSUE #4793: Research and Implement: Adding additional ADR data to the API
**What we're after:**
Several pieces of ADR data are currently missing from the API. These are the items that are still needed:

- Complainant
- Referring office
- Case status
- Non-monetary terms/non-civil penalties
- Commission votes: commissioner names and vote
- Citations (lower priority)

**Completion criteria:**
- [ ] The items listed in this ticket are added to the API

**Commission votes on EQS:** Example: ADR #1
Screen Shot 2021-04-15 at 3 47 51 PM

**Designs:**
ADR Canonical Copy 2

### PR DESCRIPTION
**Pull request by pkfec**
## Summary (required)

- Resolves #4793 

Add ADR additional parameters:
 - Complainant
-  Case status
-  Non-monetary terms/non-civil penalties
-  Commission votes: commissioner names and vote
-  Citations 

### Required reviewers

- 1 back end developer and @JonellaCulmer 

## Impacted areas of the application

-  ADR 

## How to test
- Run: `git checkout feature/4793-adr-parameters`
- Run: `pytest` (All tests PASS)
- Setup local Elasticsearch service and test on local env: Follow [Elasticsearch-7.x.0-setup-and-test-locally]( https://github.com/fecgov/openFEC/wiki/Elasticsearch-7.x.0-setup-and-test-locally)
- Delete existing indexes, run: `python cli.py delete_index docs`
- Create `docs` indexes, run:  `python cli.py create_index`
- Verify elasticsearch mappings: `http://localhost:9200/_mappings` (on browser, adr mappings will show)
- Load adr(s), run:  `python cli.py load_adrs 123` , `python cli.py load_adrs 333`,  `python cli.py load_adrs 900` or your choice
- http://localhost:9200/docs/_doc/adr_123
- http://localhost:9200/docs/_doc/adr_333
- http://localhost:9200/docs/_doc/adr_900
- compare local es data vs EQS data 

** Notes: If this PR is deployed  to `Dev` space then consider 
1. delete exiting indexes
2. create new docs index
3. load legal docs onto `Dev` space

Screenshots of ADR data on `EQS` vs `PG` vs `feature/4793-adr-parameters`:

### **Citations:**

**EQS:**

Screen Shot 2022-10-12 at 8 47 15 PM

**PG:**
Query:
```
SELECT DISTINCT VIOLATIONS.statutory_citation, VIOLATIONS.regulatory_citation, entity.name, entity.entity_id
    FROM  fecmur.VIOLATIONS, fecmur.ENTITY
    WHERE VIOLATIONS.case_id = 36000000000181
    AND VIOLATIONS.ENTITY_ID = ENTITY.ENTITY_ID
    AND (case when trim(VIOLATIONS.STAGE) = '' or VIOLATIONS.STAGE is null then 'None'
		     else upper(VIOLATIONS.STAGE)
	end ) in (select case when trim(STAGE_ORDER.STAGE) = '' or STAGE_ORDER.STAGE is null then 'None'
		                  else upper(STAGE_ORDER.STAGE)
			         end
			  from fecmur.STAGE_ORDER)
    ORDER BY entity.name

```
Screen Shot 2022-10-12 at 8 50 27 PM

**local api -> legal/docs endpoint:**
Screen Shot 2022-10-12 at 8 47 56 PM

---------------------------------------------------------------------------------------------------------------------------
---------------------------------------------------------------------------------------------------------------------------

### **Commission votes: commissioner names and vote**

**EQS:**
Screen Shot 2022-10-12 at 8 52 57 PM

**PG:**

Query:
    
```
SELECT DISTINCT c."action", c.vote_date , e.name as commissioner_name,  v.vote_type  
    FROM fecmur.COMMISSION c, fecmur.votes v, fecmur.entity e
    WHERE c.CASE_ID = 36000000000181
    AND c.commission_id = v.commission_id
    AND e.entity_id =  v.entity_id
    ORDER BY c.vote_date desc;

```
Screen Shot 2022-10-12 at 8 55 57 PM

**local api -> legal/docs endpoint:**

Screen Shot 2022-10-12 at 8 53 17 PM

----------------------------------------------------------------------------------------------------------------------------
----------------------------------------------------------------------------------------------------------------------------

### Non-monetary terms:

**EQS:**
Screen Shot 2022-10-12 at 9 03 55 PM

**PG:**
Query:

   ```
SELECT DISTINCT nt.term_description, nt.term_id
    FROM fecmur.case c, fecmur.SETTLEMENT s, fecmur.RELATEDOBJECTS r, fecmur.NON_MONETARY_TERM NT
    WHERE c.case_id  = 36000000000181
    AND c.case_id = s.case_id
    AND UPPER (SETTLEMENT_TYPE) LIKE 'PENALTY'
    AND r.RELATION_ID = 8
    AND r.MASTER_KEY = s.settlement_id
    AND R.DETAIL_KEY = NT.TERM_ID
```

Screen Shot 2022-10-12 at 9 06 47 PM

**local api -> legal/docs endpoint:**

Screen Shot 2022-10-12 at 9 04 27 PM
---------------------------------------------------------------------------------------------------------------------------
---------------------------------------------------------------------------------------------------------------------------

**Case status is populated on API based on disposition description:** 
- There is no case_status field on EQS

**local api -> legal/docs endpoint:**
Screen Shot 2022-10-17 at 10 37 16 AM

----------------------------------------------------------------------------------------------------------------------------
----------------------------------------------------------------------------------------------------------------------------

### Complainant/Referring office: 

**EQS:**

Screen Shot 2022-10-12 at 9 23 17 PM

**PG:**
Query:
```
SELECT name as complainant_name
   FROM fecmur.players 
   JOIN fecmur.role USING (role_id)
   JOIN fecmur.entity USING (entity_id)
   WHERE  players.case_id = %s
   AND players.role_id IN (100095)
```
Screen Shot 2022-10-12 at 9 24 52 PM

**local api -> legal/docs endpoint:**
Screen Shot 2022-10-12 at 9 19 09 PM

--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------

### CODE REVIEW

**Comment by fec-jli** [comment]
endpoint test urls: `/legal/docs/<doc_type>/<no>`
http://127.0.0.1:5000/v1/legal/docs/adrs/123?
http://127.0.0.1:5000/v1/legal/docs/adrs/333?
http://127.0.0.1:5000/v1/legal/docs/adrs/900?

**Comment by pkfec** [comment]
> endpoint test urls: `/legal/docs/<doc_type>/<no>` http://127.0.0.1:5000/v1/legal/docs/adrs/123? http://127.0.0.1:5000/v1/legal/docs/adrs/333? http://127.0.0.1:5000/v1/legal/docs/adrs/900?

Thanks @fec-jli

File: webservices/legal_docs/current_cases.py [hunk 1]
```diff
@@ -20,7 +19,7 @@
 logger = logging.getLogger(__name__)
 
 # for debug, uncomment this line
-# logger.setLevel(logging.DEBUG)
+logger.setLevel(logging.DEBUG)
```
Comment (fec-jli) [inline webservices/legal_docs/current_cases.py:22]: Please comment out this debug line. Thanks

Comment (pkfec) [reply webservices/legal_docs/current_cases.py]: @fec-jli debug statement is now commented and the changes are pushed to this branch.

**Review by fec-jli — APPROVED** [review]
Great job. Thanks

**Pull request merged**